### ISSUE TITLE
data processing

### ISSUE THREAD

**Issue by Gierdong** [ZYY0844/radarODE-MTL#1]
Hello, regarding the reference in your readme that 'The input size of the radarODE and radarODE MTL are the spectra with size 50x71x120 (e.g. sst_steg-0. npy)', which frequency is 71

**Comment by ZYY0844** [comment]
Hi, 
The dimension for the frequency axis is 71, so here we cover a certain range of the frequency domain (e.g., 10-25Hz).

**Comment by Gierdong** [comment]
Thank you for your answer. Could you please provide me with the specific code for data processing.

**Comment by ZYY0844** [comment]
Sure, plz find the latest MATLAB code for SST generation https://github.com/ZYY0844/radarODE-MTL/blob/091c5fe883c53482be4a0410c4a3019de210be91/MMECG_to_SST.m

**Comment by Gierdong** [comment]
Thank you very much for your answer, but it seems that there is a lack of processing methods for anchor_seg_0.npy and ecg2e-g.0.npy data

**Comment by ZYY0844** [comment]
Yes, the anchor and ECG signal are quite easy to generate, but you need some preprocessing to divide the single cardiac cycle from the ground truth ECG signal using neurokit2. 

In addition, if you are not really interested in the prediction of anchor and cycle length, I suggest you use "Given Weight" in the MTL training strategy and use the segments from ECG as the ground truth. (in this case, you will not need any preprocessing.)

If you still feel confused, plz add my Wechat "ZYY7CE" for more info.
